sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -117,15 +125,40 @@ class DatalabAppElement extends Polymer.Element {
    * We do this to lazy load pages as the user clicks them instead of letting
    * the browser pre-load all the pages on the first request.
    */
-  _pageChanged(page: string) {
+  _pageChanged(newPage: string, oldPage: string) {
     // Build the path using the page name as suffix for directory
     // and file names.
-    const subpath = 'datalab-' + page;
+    const subpath = 'datalab-' + newPage;
     const resolvedPageUrl = this.resolveUrl('../' + subpath + '/' + subpath + '.html');
```

Comment:
This might read better using string interpolation with a template literal:
  `../${subpath}/${subpath}.html`